integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": before-hook-creation
+  name: {{ template "alluxio.fullname" . }}-config
+  labels:
+    name: {{ template "alluxio.fullname" . }}-config
+    app: {{ template "alluxio.name" . }}
+    chart: {{ template "alluxio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  ALLUXIO_JAVA_OPTS: |-
+    -Dalluxio.master.hostname={{ template "alluxio.fullname" . }}-master {{ .Values.jvmOptions }} {{- if .Values.tieredstore }} -Dalluxio.worker.tieredstore.levels={{ len .Values.tieredstore.levels }} {{- range .Values.tieredstore.levels }} {{- if .alias }} -Dalluxio.worker.tieredstore.level{{ .level }}.alias={{ .alias }} {{- if .mediumtype }} -Dalluxio.worker.tieredstore.level{{ .level }}.dirs.mediumtype={{ .mediumtype }} {{- else}} -Dalluxio.worker.tieredstore.level{{ .level }}.dirs.mediumtype={{ .alias }} {{- end}} {{- end}} {{- if .path }} -Dalluxio.worker.tieredstore.level{{ .level }}.dirs.path={{ .path }} {{- end}} {{- if .quota }} -Dalluxio.worker.tieredstore.level{{ .level }}.dirs.quota={{ .quota }} {{- end}} {{- if .high }} -Dalluxio.worker.tieredstore.level{{ .level }}.watermark.high.ratio={{ .high }} {{- end}} {{- if .low }} -Dalluxio.worker.tieredstore.level{{ .level }}.watermark.low.ratio={{ .low }} {{- end}} {{- end}} {{- end}} {{- range $key, $val := .Values.properties }} -D{{ $key }}={{ $val }} {{- end}} 
+  ALLUXIO_MASTER_JAVA_OPTS: |-
+    -Dalluxio.master.web.bind.host=0.0.0.0 {{- range $key, $val := .Values.master.properties }} -D{{ $key }}={{ $val }} {{- end}} 
+  ALLUXIO_JOB_MASTER_JAVA_OPTS: |-
+    -Dalluxio.job.master.bind.host=0.0.0.0 {{- range $key, $val := .Values.jobMaster.properties }} -D{{ $key }}={{ $val }} {{- end}} 
+  ALLUXIO_WORKER_JAVA_OPTS: |-
+    -Dalluxio.worker.bind.host=0.0.0.0 {{- if eq .Values.fuse.shortCircuitPolicy "uuid" }} -Dalluxio.worker.data.server.domain.socket.address=/opt/domain -Dalluxio.worker.data.server.domain.socket.as.uuid=true {{- end}} {{- if .Values.worker.resources  }} {{- if .Values.worker.resources.requests }} {{- if .Values.worker.resources.requests.memory }} -Dalluxio.worker.memory.size={{ .Values.worker.resources.requests.memory }} {{- end}} {{- end}} {{- end}} -Dalluxio.worker.rpc.port={{ .Values.worker.resources.ports.rpc }} -Dalluxio.worker.web.port={{ .Values.worker.resources.ports.web }} {{- range $key, $val := .Values.worker.properties }} -D{{ $key }}={{ $val }} {{- end}} -Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME} 
+  ALLUXIO_JOB_WORKER_JAVA_OPTS: |-
+    -Dalluxio.job.worker.rpc.port={{ .Values.jobWorker.resources.ports.rpc }} -Dalluxio.job.worker.data.port={{ .Values.jobWorker.resources.ports.data }} -Dalluxio.job.worker.web.port={{ .Values.jobWorker.resources.ports.web }} -Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME}  
+  ALLUXIO_WORKER_TIEREDSTORE_LEVEL0_DIRS_PATH: /dev/shm
```

Comment:
Why do we have this field outstanding here?

Reply:
Do you mean the filed `ALLUXIO_WORKER_TIEREDSTORE_LEVEL0_DIRS_PATH` ? If so , please see https://github.com/Alluxio/alluxio/blob/6eda7172a4bcc5f3974ed5bcba79c35946770956/integration/docker/entrypoint.sh#L148